bokeh/models/dom.py

```diff
@@ -165,6 +192,25 @@ def __init__(self, *args, **kwargs) -> None:
     hex = Bool(default=True)
     swatch = Bool(default=True)
 
+class HTML(Model, Qualified):
+    """ """
+
+    # explicit __init__ to support Init signatures
+    def __init__(self, html: Init[str | list[str | DOMNode | UIElement]] = Intrinsic, **kwargs) -> None:
+        super().__init__(html=html, **kwargs)
+
+    html = Required(Either(String, List(Either(String, Instance(DOMNode), Instance(UIElement)))), help="""
+    Either a parsed HTML string with optional refereces to bokeh objects using
+    ``<ref id="..."></ref>`` syntax. Or a list of parsed HTML interleaved with
+    bokeh's objects. Any DOM node or UI element (event a plot) can be referenced
+    here.
```

Comment:
```suggestion
    Either of:

     * A parsed HTML string with optional references to Bokeh objects using
    ``<ref id="..."></ref>`` syntax. 

    * A list of parsed HTML interleaved with Bokeh's objects. Any DOM node or
    UI element (event a plot) can be referenced here.
```
what does "event a plot" mean?

Reply:
`even`.